napalm/base/canonical_map.py

```diff
@@ -72,6 +72,8 @@
     "Tunnel": "Tunnel",
     "Tun": "Tunnel",
     "Tu": "Tunnel",
+    "Tw": "TwoGigabitEthernet",
+    "Twe": "TwentyFiveGigE",
```

Comment:
@bbmin7b5 Wouldn't it be better to be "Two" and "Twe" as "Tw" could be ambiguous with both "TwoGigabitEthernet" and "TwentyFiveGigE"?